code/game/objects/items/weapons/grenades/chem_grenade.dm

```diff
@@ -10,6 +10,8 @@
 	var/path = 0
 	var/obj/item/device/assembly_holder/detonator = null
 	var/list/beakers = new/list()
+	var/list/temp_reagents = new/list()
```

Comment:
Just above the for loop is exactly where they should go - it's as close to their relevant code as possible, and they don't stay on the object for no reason.